omnipaxos/tests/ble_test.rs

```diff
@@ -15,12 +15,7 @@ use utils::{TestConfig, TestSystem};
 fn ble_test() {
     let cfg = TestConfig::load("ble_test").expect("Test config loaded");
 
-    let mut sys = TestSystem::with(
-        cfg.num_nodes,
-        cfg.election_timeout_ms,
-        cfg.num_threads,
-        cfg.storage_type,
-    );
+    let mut sys = TestSystem::with(cfg.clone());
```

Comment:
I think clone can be avoided if you assign the num_elections and the ms timeout as variables before.

Reply:
Maybe `TestConfig` can just implement `Copy` since its so small, only generated for tests, and keeps it so that adding new fields doesn't require changing the start of every single test.